scripts/pkg/.goreleaser.yml.template

```diff
@@ -1,22 +1,32 @@
 builds:
   - id: linux
-    binary: var/db/newrelic-infra/newrelic-integractions/bin/nri-{{.Env.NAME}}
+    binary: var/db/newrelic-infra/newrelic-integrations/bin/nri-{{.Env.NAME}}
     dir: nri-config-generator
     ldflags:
       - -X main.integration={{.Env.NAME}} -X main.integrationVersion=${.Tag} -X main.gitCommit={.Commit} -X main.BuildDate=${.Date}
     goos:
       - linux
-    # This field is automatically populated from the exporter.yaml
-    goarch: []
+    goarch: []  # This field is automatically populated from the exporter.yaml
+    hooks:
+      # This is ugly but needed.
```

Comment:
if we are calling goreleaser from a bash script, would make sense to add this logic in the script and have everything there?